assets/templates/partials/release/status-header.tmpl

```diff
@@ -1,4 +1,27 @@
 <div>
+  <div class="ons-u-pb-s">
+    {{ if .Description.Census2021 }}
+      <div
+        {{ if or (eq .PublicationState.Type "upcoming") (eq .PublicationState.Type "cancelled") }}
+          class="ons-u-pb-m"
+        {{ end }}
```

Comment:
Is this still required? My understanding is that this class is needed to add a gap with the "this release is not yet published" div but with this change that one will not be the next element.

Reply:
yes you are correct. I have fix this.